bundles/org.openhab.binding.chatgpt/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,8 @@
 	<description>This binding allows interaction with OpenAI's ChatGPT.</description>
 	<connection>cloud</connection>
 
+	<service-id>org.openhab.voice.chatgpthli</service-id>
+
+	<config-description-ref uri="voice:chatgpthli"/>
```

Comment:
Have not seen this before, but im also not familiair with voice related to this addon file, so just a comment as reminder to check.